chromadb/test/property/strategies.py

```diff
@@ -325,8 +345,8 @@ def recordsets(
 # enable/disable rules. Disabled rules cause the entire test to be marked invalida and,
 # combined with the complexity of our other strategies, leads to an
 # unacceptably increased incidence of hypothesis.errors.Unsatisfiable.
-class DeterministicRuleStrategy(SearchStrategy):
-    def __init__(self, machine):
+class DeterministicRuleStrategy(SearchStrategy):  # type: ignore
```

Comment:
@levand PTAL here, it seems some of the things here rely on internal hypothesis types - not sure if this is correct.